src/core/public/notifications/notifications_service.mock.ts

```diff
@@ -23,7 +23,7 @@ import {
 } from './notifications_service';
 import { toastsServiceMock } from './toasts/toasts_service.mock';
 
-type DeeplyMocked<T> = { [P in keyof T]: jest.Mocked<T[P]> };
+export type DeeplyMocked<T> = { [P in keyof T]: jest.Mocked<T[P]> };
```

Comment:
Actually you can remove this completely and use the globally available `MockedKeys` defined in `typings/index.d.ts`